pyresample/utils/proj4.py

```diff
@@ -64,7 +64,7 @@ def proj4_dict_to_str(proj4_dict, sort=False):
     params = []
     for key, val in items:
         key = str(key) if key.startswith('+') else '+' + str(key)
-        if key in ['+no_defs', '+no_off', '+no_rot']:
+        if key in ['+no_defs', '+no_off', '+no_rot'] or val is None:
```

Comment:
I just realized, is there any point to the `key in [...]` check anymore if we assume that a value will be `None` for value-less parameters?

Reply:
I guess it depends on whether we assume that `no_{defs,off,rot}` keys will always have a value of `None` in `proj4_dict`.

I would think this is a safe assumption, but leaned towards keeping the `key in ...` check out of caution.

Reply:
Ok, let's keep it as is. I think it would be save to drop it, but given the plans to drop this altogether I'm not sure there is a point. Anyway, pyproj puts a `None` in the dict for `no_defs` so it seems like a safe assumption.